src/Common/memcpySmall.h

```diff
@@ -27,18 +23,62 @@
   * Use with caution.
   */
 
+#if defined(__AVX512F__) && !defined(MEMORY_SANITIZER)
```

Comment:
Missing comment, why `!defined(MEMORY_SANITIZER)` here?
And why it is not required below, in `#elif defined(__SSE2__)`?